main/mariadb/APKBUILD

```diff
@@ -5,14 +5,15 @@
 # Contributor: TBK <alpine@jjtc.eu>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=mariadb
-pkgver=10.2.15
+pkgver=10.3.7
 pkgrel=0
 pkgdesc="A fast SQL database server"
-url="http://www.mariadb.org"
+url="https://www.mariadb.org"
 pkgusers="mysql"
 pkggroups="mysql"
 arch="all"
 license="GPL-2.0-or-later"
+#options="!check" # dbug test fails under rootbld
```

Comment:
Why do you comment out this instead of removing?

Reply:
I did not comment it out. I moved it. Not sure why it was left in from this commit https://github.com/alpinelinux/aports/commit/9f6c5c776a20c9ff350aa839795d1c2bf65fb0ac